x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/search/SourceGenerator.java

```diff
@@ -45,10 +45,7 @@ public static SearchSourceBuilder sourceBuilder(QueryContainer container, QueryB
         // Iterate through all the columns requested, collecting the fields that
         // need to be retrieved from the result documents
 
-        // NB: the sortBuilder takes care of eliminating duplicates
-        container.fields().forEach(f -> f.v1().collectFields(sortBuilder));
-        sortBuilder.build(source);
```

Comment:
The `sortBuilder` is still defined above. Needs removal? Also, the two comments above are not needed anymore.